Signal/src/ViewControllers/FingerprintViewController.m

```diff
@@ -97,6 +129,7 @@ - (void)configureWithRecipientId:(NSString *)recipientId
     OWSRecipientIdentity *_Nullable recipientIdentity =
         [[OWSIdentityManager sharedManager] recipientIdentityForRecipientId:recipientId];
     OWSAssert(recipientIdentity);
+    self.identityKey = recipientIdentity.identityKey;
```

Comment:
By capturing the identity key when we enter these views, we prevent the edge case where the user verifies a key that we learned about while this view was open.